packages/fuels-abigen-macro/tests/harness.rs

```diff
@@ -1549,55 +1430,3 @@ async fn workflow_struct_inside_enum() {
     let expected = Shaker::Cosmopolitan(Recipe { ice: 22, sugar: 99 });
     assert_eq!(result.value, expected);
 }
-
-#[tokio::test]
-async fn test_logd_receipts() {
```

Comment:
Why are we deleting this test case?

Reply:
I didn't restore this because I'm not sure why it was deleted.